wordpress-mobile/WordPress-iOS
# Login: Wires up 'connect another' button on epilogue screen.

Refs #7272 

To test:
On the epilogue screen, 
- confirm clicking the continue button dismisses the screen. 
- confirm clicking the "add another" button shows the site address view controller
- confirm the site address view controller has a cancel button
- confirm that the correct button title is shown when there are zero blogs, and when there is one or more blogs.

Needs review: @nheagy 


![simulator screen shot jul 11 2017 5 47 45 pm](https://user-images.githubusercontent.com/1435271/28094247-3213cf6c-6661-11e7-96e3-e836e18f7825.png)
